md/bdm-access-control.md

```diff
@@ -134,4 +138,174 @@ Deploy your access control file.
 
 Access to your previous data is now controlled by the BDM Access Control file you just deployed. A novice member of HR can't see the attribute *medicalComment* on a LeaveRequest anymore.  
 Connect onto the portal as a user with the profile *Novice HR*. Go to your application: the medical comment is always empty.  
-Connect now as a user with the profile *Experimented HR*. On your application, the medical comments is displayed.
+Connect now as a user with the profile *Experimented HR*. On your application, the medical comments is displayed.  
+
+### II - Invoice
+
+**Business use case:** A company has to handle invoices. An invoice references an order and a customer.  
+An employee in charge of the preparation of the order shall access to the order but not to the customer of an invoice.  
+An order is represented by several invoice line. An invoice line is made of a product , and a quantity. A product is defined by a name and a price. An employee in charge of the preparation of the order shall access to the product and the quantity but not to the price of a product.  
+A sales of the company shall access to both the customer and the order of an invoice.  
+A customer is represented by a name, an email and an address. A sells shall access to the name and the email of the customer, but not to its address.  
+
+**Technical use case:** Grant access to some complex attributes (with composition or aggregation relations) of a business object depending on the profiles of the requester. The access control for the business object with an aggregation / composition relation has to be defined too.  
+
+**1. BDM definition**
+
+Define a *Customer* business object, with the following attributes:  
+
+| Name | Type |
+|---|---|
+| name | STRING |
+| email | STRING |
+| address | STRING |
+
+Define a *Product* business object, with the following attributes:  
+
+| Name | Type |
+|---|---|
+| name | STRING |
+| price | INTEGER |
+
+Define a *InvoiceLine* business object, with the following attributes:  
+
+| Name | Type | Relation | Eager |
+|---|---|---|---|
+| product | Product | Composition | true |
+| quantity | INTEGER |
+
+Define a *Invoice* business object, with the following attributes:  
+
+| Name | Type | Multiple | Relation | Eager |
+|---|---|---|---|---|
+| customer | Customer | false | Aggregation | true |
+| fullOrder | InvoiceLine | yes | Composition | true |
+| orderDate | DATE ONLY | false |  |
+
```

Comment:
Is the BDM automatically deploy? or we need a reminder to "Deploy the BDM"?

Reply:
it is automatically deployed when you leave the wizard. There is no way to deploy manually the bdm from the studio.